blueocean-rest/src/main/java/io/jenkins/blueocean/rest/model/BlueRun.java

```diff
@@ -257,6 +265,45 @@
         public abstract Object getCause();
     }
 
+    @ExportedBean
+    public static final class BlueDownstreamRun {
+        private final String downstreamProject;
+        private final int downstreamBuild;
+
+        public BlueDownstreamRun(String downstreamProject, int downstreamBuild) {
+            this.downstreamProject = downstreamProject;
+            this.downstreamBuild = downstreamBuild;
+        }
+
+        @Exported(name="downstreamProject")
```

Comment:
Gratuitous `name`—this would be the default anyway.

Reply:
It seems that way to me also, but it matches the rest of the code in that file so I'm going for consistency.